src/OpenTracing/Extractor.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace OpenTracing;
+
+/**
+ * An Extractor extracts Span instances from a format-specific "carrier" object.
+ *
+ * Typically the carrier has just arrived across a process boundary, often via
+ * an RPC (though message queues and other IPC mechanisms are also reasonable
+ * places to use an Extractor).
+ *
+ * See Injector and Tracer.extractor().
+ *
+ * @package OpenTracing
+ */
+interface Extractor {
```

Comment:
We put Extractor and Injector into a single file in .py and I thought that made sense there – not sure if PHP usually does one public class per file or what.